plan/errors.go

```diff
@@ -75,6 +76,7 @@ var (
 	ErrDupFieldName         = terror.ClassOptimizer.New(codeDupFieldName, mysql.MySQLErrName[mysql.ErrDupFieldName])
 	ErrNonUpdatableTable    = terror.ClassOptimizer.New(codeNonUpdatableTable, mysql.MySQLErrName[mysql.ErrNonUpdatableTable])
 	ErrInternal             = terror.ClassOptimizer.New(codeInternal, mysql.MySQLErrName[mysql.ErrInternal])
+	ErrNonUniqTable         = terror.ClassOptimizer.New(codeNonUniqTable, "Not unique table/alias: '%s'")
```

Comment:
Can we use `mysql.MySQLErrName[mysql. ErrNonuniqTable]`?

Reply:
yep